rotkehlchen/tests/unit/test_avalanche_manager.py

```diff
@@ -1,11 +1,29 @@
+from unittest.mock import patch
+
+from flaky import flaky
+
+from rotkehlchen.types import deserialize_evm_tx_hash
+
+
+@flaky(max_runs=5, min_passes=1)  # covalent api might be really flaky
 def test_get_transaction_receipt(avalanche_manager):
-    tx_hash = '0x7e7dee4b821331437524d0fd176a5090abbe4e857c849b06dfe9224f00e22f4d'
-    result = avalanche_manager.get_transaction_receipt(
-        '0x7e7dee4b821331437524d0fd176a5090abbe4e857c849b06dfe9224f00e22f4d',
+    def mock_covalent_get_transaction_receipt(_tx_hash):
+        return None  # this means no result from covalent
+
+    covalent_patch = patch.object(
+        avalanche_manager.covalent,
+        'get_transaction_receipt',
+        side_effect=mock_covalent_get_transaction_receipt,
```

Comment:
We use this somewhere in the code. Left doesn't like functions inside functions + is just a no op

```suggestion
        side_effect=lambda *args: None,
```

Reply:
Okay, I also remember that in some other pt you and Left mentioned that lambda functions are not good. But okay, will adjust